tests/Features/ReturnTypes/AppAccessTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Features\ReturnTypes;
+
+use function PHPStan\Testing\assertType;
+
+class AppAccessTest
+{
+    public function testEnvironment(): void
+    {
+        assertType('string', app()->environment());
+        assertType('bool', app()->environment('local'));
+        assertType('bool', app()->environment(['local', 'production']));
+    }
+}
```

Comment:
Instead of this file can you add a file to `tests/type/data` and the appropriate entry in the `GeneralTypeTest` ?